test/WebResource.js

```diff
@@ -0,0 +1,116 @@
+import * as helpers from './helpers';
+import * as chai from 'chai';
+import * as chaiDateTime from 'chai-datetime';
+
+chai.use(chaiDateTime);
+const { expect } = chai;
+
+const hex = '5261777221';
+const buf = Buffer.from(hex, 'hex');
+
+const webResource = {
+	filename: 'logo.png',
+	href: 'test',
+	contentType: 'image/png',
+	contentDisposition: null,
+	size: buf.byteLength,
+};
```

Comment:
Should this be part of the `before` block?

Reply:
@fisehara It can be done on a before or even better `beforeAll` hook, but I don't see how this becomes better: to achieve that, we will have to add the variables hex buf and webResource to the helpers.describe with a `let` and then only get them a type on the before method, which will lead to something constant to be created with a let. I did however changed this to be inside the describe block as this follow the pattern used on other test resources on this repo.